components/feedstock/links.js

```diff
@@ -14,7 +18,7 @@ export const Links = ({ links, doi_citation }) => {
         Links:
       </Text>
 
-      <LinkGroup size='xs' members={allLinks} sx={{ ml: 4 }} />
+      <LinkGroup members={allLinks} sx={{ ml: 4 }} />
```

Comment:
@Shane98c / @katamartin, does the `LinkGroup` component allow specifying `target` or `rel` ? I'm trying to make links open in a new tab. i couldn't find this information by looking at the codebase/docs.

Reply:
it doesn't look like we pass other props through to the Button component that LinkGroup is using, so at the moment that's not possible. It wouldn't be a major change to add that functionality though!

Reply:
it doesn't look like we pass other props through to the Button component that LinkGroup is using, so at the moment that's not possible. It wouldn't be a major change to add that functionality though!